File: go/raindrops/raindrops.go
// Package provides a fizzbuzz puzzle with a twist
package raindrops

import "fmt"

// Convert parses a number into raindrops on the tin roof
func Convert(number int) string {
	str := ""
	if number%3 == 0 {
		str += "Pling"
	}
	if number%5 == 0 {
		str += "Plang"
	}
	if number%7 == 0 {
		str += "Plong"
	}
	if str == "" {
		return fmt.Sprintf("%d", number)
	}
	return str
}

File: python/raindrops/raindrops.py
def convert(number):
    s = ""
    if number % 3 and number % 5 and number % 7 != 0:
        s += str(number)
    if number % 3 == 0:
        s += "Pling"
    if number % 5 == 0:
        s += "Plang"
    if number % 7 == 0:
        s += "Plong"
    return s
